src/frontend/Axis.hpp

```diff
@@ -219,6 +223,22 @@ class Axis : public Drawable {
   /// gets the number of ticks on this axis
   Vector<int, 2> get_ticks() const { return {m_nx_ticks, m_ny_ticks}; }
 
+  /// get the number of maximum yticks length (in pixels)
+  int max_ytick_pixels() const {
```

Comment:
This calculation should really be done by the `Style` class, but since the calculation is still not correct please put it inline in `Axis::draw_common_ylabel` along with a TODO comment